### ISSUE TITLE
Epic: Product: Accessibility Audit and Fixes

### ISSUE THREAD

**Issue by Rabia2219** [hackforla/tdm-calculator#3409]
### Overview
We need to organize and finish the accessibility project so that we can deliver a fully accessible site to the stakeholders on turnover.

### Action Items
- [x] Find all accessibility related issues
- [x] Create an outline
- [x] Organize them into a hierarchal structure
- [x] Identify any remaining work needed on the accessibility repo to support the TDM launch
- [ ] Review with product and dev. Update if needed based on feedback

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, ISSUE NUMBER]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

**Comment by Rabia2219** [comment]
### Open Issue

#### Epics
- https://github.com/hackforla/tdm-calculator/issues/2898
- https://github.com/hackforla/tdm-calculator/issues/2897
- https://github.com/hackforla/tdm-calculator/issues/2896
- https://github.com/hackforla/tdm-calculator/issues/2895
- https://github.com/hackforla/tdm-calculator/issues/2894
- https://github.com/hackforla/tdm-calculator/issues/2893
- https://github.com/hackforla/tdm-calculator/issues/2892
- https://github.com/hackforla/tdm-calculator/issues/2891

#### Audit issues
- None

#### Fix issues
- https://github.com/hackforla/tdm-calculator/issues/3046
- https://github.com/hackforla/tdm-calculator/issues/3044
- https://github.com/hackforla/tdm-calculator/issues/3041
- https://github.com/hackforla/tdm-calculator/issues/3036


#### Not audit or fix issues that have the 'feature: Accessibility' label
- https://github.com/hackforla/tdm-calculator/issues/3388
- https://github.com/hackforla/tdm-calculator/issues/3381
- https://github.com/hackforla/tdm-calculator/issues/3372
- https://github.com/hackforla/tdm-calculator/issues/3227

**Comment by Rabia2219** [comment]
#### Hierarchy
- https://github.com/hackforla/tdm-calculator/issues/2891
  - https://github.com/hackforla/tdm-calculator/issues/3046
- https://github.com/hackforla/tdm-calculator/issues/2892
  - https://github.com/hackforla/tdm-calculator/issues/3040
  - https://github.com/hackforla/tdm-calculator/issues/3041
- https://github.com/hackforla/tdm-calculator/issues/2893
  - https://github.com/hackforla/tdm-calculator/issues/3044
- https://github.com/hackforla/tdm-calculator/issues/2894
   - https://github.com/hackforla/tdm-calculator/issues/3036
- https://github.com/hackforla/tdm-calculator/issues/2895
- https://github.com/hackforla/tdm-calculator/issues/2896
   - https://github.com/hackforla/tdm-calculator/issues/3037
   - https://github.com/hackforla/tdm-calculator/issues/3038
   - https://github.com/hackforla/tdm-calculator/issues/3039
- https://github.com/hackforla/tdm-calculator/issues/2897
  - https://github.com/hackforla/tdm-calculator/issues/2960
- https://github.com/hackforla/tdm-calculator/issues/2898

**Comment by Rabia2219** [comment]
## Remaining work related to TDM finishing its accessibility project
- [x] https://github.com/hackforla/accessibility/issues/64

## Remaining work for accessibility repo, does not impact TDM finishing
- [ ] https://github.com/hackforla/accessibility/issues/32
- [x] Create new issues
    - [x] Update all pages in the solutions index to add icons at the top
       - new issue https://github.com/hackforla/accessibility/issues/66
    - [x] Update all pages in the solutions index to rename TDM or Project Name Error guidance to Error Fix Example(s) from Project Teams
       - https://github.com/hackforla/accessibility/issues/67
    - [x] Make any missing pages for aria, feature and structural elements guidance from WAVE
       - https://github.com/hackforla/accessibility/issues/68
- [x] Update https://hackforla.github.io/accessibility/getting-started/perform-wave-audit/
   - [x] did not render properly, fix
